cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -4,12 +4,6 @@
 // Package bicep contains an implementation of provider.Provider for Bicep. This
 // provider is registered for use when this package is imported, and can be imported for
 // side effects only to register the provider, e.g.:
-//
```

Comment:
NIT: you can delete env more of the pre-amble of this comment, we no longer register the provider on import anymore.